service/verifier/verifier_server_test.go

```diff
@@ -40,8 +40,8 @@ const (
 type cryptoParameters struct {
 	cryptoPath   string
 	update       *servicepb.VerifierUpdates
-	metaTxSigner *sigtest.NsEndorser
-	dataTxSigner *sigtest.NsEndorser
+	metaTxSigner *sigtest.TxNsEndorser
+	dataTxSigner *sigtest.TxNsEndorser
```

Comment:
`metaTxEndorser` and `dataTxEndorser` instead of `metaTxSigner` and `dataTxSigner`.